modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/dht/IgniteCacheTopologySplitAbstractTest.java

```diff
@@ -217,14 +217,13 @@ protected void checkSegmented(InetSocketAddress sockAddr, long timeout) throws S
 
         /** {@inheritDoc} */
         @Override protected void writeToSocket(
-            Socket sock,
-            TcpDiscoveryAbstractMessage msg,
-            byte[] data,
-            long timeout
+                Socket sock,
+                byte[] data,
```

Comment:
@aleksey-kurinov , this change looks like extra tabulation.

Reply:
fixed